sdk/communication/AzureCommunicationChat/Source/Generated/AzureCommunicationChatService.swift

```diff
@@ -1750,23 +1775,29 @@ public final class AzureCommunicationChatService {
     public func create(
         chatThread: CreateChatThreadRequest,
         withOptions options: CreateChatThreadOptions? = nil,
-        completionHandler: @escaping HTTPResultHandler<MultiStatusResponse>
+        completionHandler: @escaping HTTPResultHandler<ChatThread>
     ) {
         // Construct URL
         let urlTemplate = "/chat/threads"
         let pathParams = [
             "$endpoint": client.endpoint.absoluteString
         ]
-
         // Construct query
         let queryParams: [QueryParameter] = [
-            ("api-version", "2020-09-21-preview2")
+            ("api-version", "2020-11-01-preview3")
         ]
 
         // Construct headers
         var headers = HTTPHeaders()
         headers["Content-Type"] = "application/json"
         headers["Accept"] = "application/json"
+        // Process endpoint options
+        // Query options
+
+        // Header options
+        if let azureAcsIdempotencyKey = options?.azureAcsIdempotencyKey {
```

Comment:
I think this was pretty much the only new manual fix I had to add (rest were just re-applying previous manual fixes). Generated code was producing:

`if let azure - Acs - IdempotencyKey = options?.azure - Acs - IdempotencyKey {
            headers["azure-Acs-IdempotencyKey"] = azureAcsIdempotencyKey
        }`

Reply:
@sacheu can you take a look? I thought we'd squashed most of these.

Reply:
I have not tried using preview 3 chat sdk. I will take a look at this issue.

Reply:
Thanks @sacheu for the fixes, only new thing I notice is the api version causes linting error with the underscores:

Generated is producing:
        `case v2020_11_01_preview3 = "2020-11-01-preview3"`

Regenerated SDK has been tested with demo app and looks good otherwise.